docs/tutorial/part-five/index.md

```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/) before continuing here.
+> हे ट्यूटोरियल गॅट्सबीच्या डेटा लेयर बद्दलच्या मालिकेचा एक भाग आहे. आपण जात असल्याचे सुनिश्चित करा [part 4](/tutorial/part-four/) येथे सुरू ठेवण्यापूर्वी.
 
-## What's in this tutorial?
+## या ट्यूटोरियल मध्ये काय आहे?
 
-In this tutorial, you'll be learning about how to pull data into your Gatsby site using GraphQL and source plugins. Before you learn about these plugins, however, you'll want to know how to use something called GraphiQL, a tool that helps you structure your queries correctly.
+या ट्यूटोरियल मध्ये, आपण ग्राफिक्स आणि स्त्रोत प्लगइन वापरुन आपल्या गॅटस्बी साइटमध्ये डेटा कसा काढायचा याबद्दल शिकत आहात. तथापि आपण या प्लगइन्सबद्दल जाणून घेण्यापूर्वी, आपल्याला ग्राफिक नावाचे काहीतरी कसे वापरावे हे जाणून घ्यायचे आहे, जे आपले क्वेरी अचूकपणे रचण्यात मदत करते.
```

Comment:
ग्राफिक -> GraphiQL

 a tool that helps you structure your queries correctly. -> एक साधन जे आपणास आपल्या क्वेरीची योग्यरित्या रचना करण्यात मदत करते.

Reply:
modified